conformance_tests/sysman/test_sysman_pci/src/test_sysman_pci.cpp

```diff
@@ -62,11 +67,11 @@ TEST_F(PCI_TEST,
     EXPECT_LT(pciProps.address.device, MAX_DEVICES_PER_BUS);
     EXPECT_GE(pciProps.address.function, 0);
     EXPECT_LT(pciProps.address.function, MAX_FUNCTIONS_PER_DEVICE);
-    EXPECT_GE(pciProps.maxSpeed.gen, 0);
+    EXPECT_GE(pciProps.maxSpeed.gen, -1);
```

Comment:
If these are -1, It's an issue right ?

Reply:
Valid values are seen in discrete platform. Since the spec says it can be -1 aswell if unknown, the expectation is set to GE -1. This is seen in integrated platforms